libs/service-portal/icelandic-names-registry/src/components/EditForm/EditForm.tsx

```diff
@@ -0,0 +1,364 @@
+                        errorMessage={errors?.icelandicName?.message}
+                        backgroundColor="blue"
+                        onChange={(e) => onChange(e.target.value)}
+                        disabled={inputsDisabled}
+                      />
+                    )
+                  }}
+                />
+              </GridColumn>
+            </GridRow>
+
+            <GridRow>
+              <GridColumn
+                span={['12/12', '12/12', '6/12']}
+                paddingBottom={spacing}
+              >
+                <Controller
+                  name="type"
+                  render={({ onChange, value, name }) => {
+                    return (
+                      <Select
+                        name={name}
+                        options={nameTypeOptions}
+                        label="Tegund nafns"
+                        placeholder="Veldu tegund"
+                        value={nameTypeOptions.find(
+                          (option) => option.value === value,
+                        )}
+                        onChange={(option) =>
+                          onChange(String((option as Option).value))
+                        }
+                        size="sm"
+                        hasError={Boolean(errors?.type?.message)}
+                        errorMessage={errors?.type?.message}
+                        backgroundColor="blue"
+                        disabled={inputsDisabled}
+                        required
+                      />
+                    )
+                  }}
+                />
+              </GridColumn>
+              <GridColumn
+                span={['12/12', '12/12', '6/12']}
+                paddingBottom={spacing}
+              >
+                <Controller
+                  name="status"
+                  render={({ onChange, value, name }) => {
+                    return (
+                      <Select
+                        name={name}
+                        options={statusTypeOptions}
+                        label="Staða"
+                        placeholder="Veldu stöðu"
+                        value={statusTypeOptions.find(
+                          (option) => option.value === value,
+                        )}
+                        onChange={(option) =>
+                          onChange(String((option as Option).value))
+                        }
+                        size="sm"
+                        hasError={Boolean(errors?.status?.message)}
+                        errorMessage={errors?.status?.message}
+                        backgroundColor="blue"
+                        disabled={inputsDisabled}
+                        required
+                      />
+                    )
+                  }}
+                />
+              </GridColumn>
+            </GridRow>
+
+            <GridRow marginTop={3}>
+              <GridColumn span={'12/12'} paddingBottom={spacing}>
```

Comment:
unnecessary grid row and column? maybe just use box with margin padding?

Reply:
this applies to all cases where there is a single column with span 1

Reply:
I need to do changes to the layout. Will change this as well.